src/main/java/unicash/model/UniCash.java

```diff
@@ -89,6 +91,42 @@ public void removeTransaction(Transaction key) {
         transactions.remove(key);
     }
 
+    /**
+     * Returns the amount for each category of expenses.
+     * Note: This function ignores all 'income' transactions
+     */
+    HashMap<String, Double> getSumOfExpensePerCategory() {
+        HashMap<String, Double> sumPerCategory = new HashMap<>();
+        ObservableList<Transaction> allTransactions = getTransactionList();
+        String uncategorizedCategoryName = "Uncategorized";
+        for (Transaction t : allTransactions) {
+            if (t.getType().toString().equals("income")) {
```

Comment:
I thought of this too. However, given that the summary feature still has the line graph remaining, I doubt that I will be able to implement this feature by v1.3b. Perhaps during our weekly meeting today, if someone is lacking tasks to do for the upcoming week, they can try looking into this. Otherwise, we can consider chucking this as a feature that is [coming soon]!

Reply:
Ah okok that makes sense! We can consider making the line graph first, and if time permits, make a copy of that and change the type parameters to `income`, I think given the generality of the code you have already implemented it would be a lot easier to implement!